fog/sig/README.md

```diff
@@ -0,0 +1,53 @@
+This code (and the associated crates bundled here) provide support for the *Fog Signature Scheme*.
+
+## This Crate
+
+This crate provides a `Verifier` trait and an implementation for [`PublicAddress`](mc_account_keys::PublicAddress). This trait contains a single method, `verify_fog_sig()`, which is used to verify a user has signed off on a list of Attestation Verfiication Reports for ingest enclaves.
+
+## In Context
+
+Fog works by having a sender encrypt a transaction's destination public address using a cryptobox-like scheme based on Ristretto255 and AES-256-GCM. The sender looks up a destination user's fog server, determines the appropriate ingest enclave's Ristretto public key, and encrypts the recipient's public address for that enclave.
+
+This family of crates provides primatives to authenticate that a particular user has delegated fog operations (via a chain of trust) to a list of particular ingest server public keys.
```

Comment:
#### proposed clarification

This family of crates provides primitives to authenticate that a particular user has delegated fog operations to a list of particular ingest server public keys. Specifically, that the user signed off on a Fog operator's root certificate, that there is a certificate chain between that root and some ingest server AVRs, and that those AVRs are valid.